provisioning/roles/chrome/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+- name: Download latest chrome
+  get_url: url={{ chrome_download_url }} dest=/tmp/chrome.deb
+  sudo: yes
+
+- name: Install Chrome
+  apt: deb=/tmp/chrome.deb state=present
```

Comment:
You can skip the package download by using the URL directly here: `apt: deb={{ chrome_download_url }}`. See http://docs.ansible.com/ansible/apt_module.html